core/src/main/java/org/arquillian/cube/q/core/InstallProxy.java

```diff
@@ -26,15 +26,17 @@ public void install(@Observes(precedence = 100) CubeDockerConfiguration configur
         
         DockerCompositions cubes = configuration.getDockerContainersContent();
         ProxyManager installer = serviceLoaderInst.get().onlyOne(ProxyManager.class);
-        
-        Proxy proxy = installer.install(cubes);
-        proxyInst.set(proxy);
 
-        final CubeContainer cube = proxy.getCube();
-        cubes.add(proxy.getName(), cube);
+        if (installer != null) {
+            Proxy proxy = installer.install(cubes);
+            proxyInst.set(proxy);
+
+            final CubeContainer cube = proxy.getCube();
+            cubes.add(proxy.getName(), cube);
 
-        System.out.println("PROXY INSTALLED");
-        System.out.println(ConfigUtil.dump(cubes));
+            System.out.println("PROXY INSTALLED");
```

Comment:
Hmm really? I mean we can always use JUL which is far from ideal, but at least kinda works. /cc @aslakknutsen